### DISCUSSION TITLE
How to use SVG icons in polybar

### DISCUSSION THREAD

**Discussion by yumyumb612** [polybar/polybar#2986] [Q&A] [answered]
hoping anyone knows how to insert svg icon in polybar, can't find any solution. lol lil help? thanks :D

**Comment by patrick96** [comment, marked as answer]
Polybar is text only, no images. Technically, you could create a font from the images you want and use that font to display the "images". But not sure how feasible that is.

**Reply by yumyumb612** [reply]
ohh.. well that sucks. lol, anyways, thanks :D
